### ISSUE TITLE
Offer GBIF annotated dwca under dataset download

### ISSUE THREAD

**Issue by peterdesmet** [gbif/portal16#360]
As mentioned in https://github.com/gbif/portal16/issues/219#event-1133399327, I like the fact that the original dwca can be downloaded via the download button on the dataset page. However, what is listed in the dropdown menu could be improved (maybe you are already planning to do so):

<img width="265" alt="screen shot 2017-06-26 at 14 09 54" src="https://user-images.githubusercontent.com/600993/27538614-269df9a8-5a79-11e7-93a0-5a495ea8065e.png">

1. Maybe rephrase to "Source archive" (to indicate that it is the original one?)
2. Would add "GBIF annotated archive" => triggers download or requests to login before trigger download
3. The "GBIF annotated meta data (EML)" -> This now opens the EML in your browser. Not sure this is that useful?

Notes: the format `EML` is mentioned for the metadata. Should we mention `DWCA ARCHIVE` for the data? (cf. [at bottom of page](https://demo.gbif.org/dataset/1f968e89-ca96-4065-91a5-4858e736b5aa#dataDescription)). Also, is it `meta data` or `metadata`?

**Comment by MortenHofft** [comment]
Nice an detailed feedback - thanks

1) i have renamed it
2) should already be there (when there is occurrences to download) but named Processed occurrences. I have renamed it to GBIF annotated archive.
3) I believe someone asked for it, but I might just have invented that need. Thing is that we have it currently, and the main feedback I get is: "we used to have this, not having it is a regression" - so i'm getting a bit cautious about removing information.

*spelling mistakes*
Metadata vs meta data. I'll leave that to those who actually speak english. I have updated "meta data" in the translation file to one word (since that is what google suggested me). Currently wording is mostly just me as communications team haven't had time to go through it all. But as we move everything into the translation file and start adding more languages i believe they will proof read and change all of my danglish.

*archive format*
I have added the format to published archive, but i'm not sure it helps much seeing it. Feels more like clutter. But might be useful and i guess that is worth more :)

![screen shot 2017-07-04 at 14 14 01](https://user-images.githubusercontent.com/6759267/27829945-66941e60-60c4-11e7-8548-9e7bbbf00809.png)

It will be available in the next deploy.
